client/src/components/ReviewTile.js

```diff
@@ -1,11 +1,15 @@
 import React from "react";
 
-const ReviewTile = ({ title, rating, content }) => {
+const ReviewTile = ({ id, title, rating, content, deleteReview }) => {
+
   return (
     <div>
       <h4>{title}</h4>
       <h5>{rating}</h5>
       <p>{content}</p>
+      <div onClick={() => { deleteReview(id) }}>
+        <input type="submit" value="Delete Review"/>
```

Comment:
we could instead make this a button and move the `onClick` to that element as well - I'd say though the key piece is putting the synthetic click event as close to the actual thing that's being interacted with